cues: {
    # Rendering is done back to front.
    grid: {
        grid: {
            shape: 'grid',
            theta: 0.0,
            phi: 90.0,
            fade_in_ms: 0.0,
            fade_out_ms: 0.0,
            visible: false,
        }
    },

    dots: {
        dot: {
            repeat: 500,
            shape: 'dot',
            fade_in_ms: 5000.0,
            fade_out_ms: 5000.0,
            visible: true,
            cycle_probability: 0.005,
        },
    },
}
